hail/python/hailtop/auth/auth.py

```diff
@@ -34,9 +34,9 @@ def namespace_auth_headers(deploy_config, ns, authorize_target=True):
     return headers
 
 
-def service_auth_headers(deploy_config, service, authorize_target=True):
+def service_auth_headers(deploy_config, service, authorize_target=True, _token_file=None):
     ns = deploy_config.service_ns(service)
-    return namespace_auth_headers(deploy_config, ns, authorize_target)
+    return namespace_auth_headers(deploy_config, ns, authorize_target, _token_file)
```

Comment:
I think it's a good idea to make these kwargs like `_token_file=_token_file`. We've run into problems in the past where the arguments get out of order for the kwargs ones.